packages/parallel-workers/src/website-scan-result-aggregator.spec.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should this be website-scan-result-aggregator-worker.spec.ts to match the .ts file?

Reply:
Renamed